### PR TITLE
Updated drone yaml for drone 1.0

### PR DESCRIPTION
**Pull request by SniperSister**
### Summary of Changes
Updated Drone YAML file  for new new drone version

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -40,22 +41,10 @@ pipeline:
       - if [ $DRONE_REPO_OWNER != 'joomla' ]; then echo "The analysis check only run on the main repos"; exit 0; fi
       - rips-cli rips:scan:start -a 1 -t 1 -p $(pwd) -t 1 -R -k -T $DRONE_REPO_OWNER-$DRONE_BRANCH ||  { echo "Please contact the security team at security@joomla.org"; exit 1; }
 
-  analysis4x:
```
Comment (wilsonge) [inline .drone.yml:43]: Please leave this in ❤️

Comment (SniperSister) [reply .drone.yml:43]: In the staging branch?

**Comment by SniperSister** [comment]
@Hackwar please verify!

**Comment by SniperSister** [comment]
@HLeithner please check

**Pull request merged**